static/js/public/snap-details/exapandable-distr-chart.js

```diff
@@ -1,27 +1,21 @@
 export default function initExpandableDistroChart() {
```

Comment:
this file is now exactly the same as `expandable-details`? Why not just define it once and reuse?